spec/rubocop/cop/style/method_name_spec.rb

```diff
@@ -49,6 +49,16 @@ def +(other)
       END
     end
 
+    it 'accepts unary operator definitions' do
+      expect_no_offenses(<<-END.strip_indent)
```

Comment:
The other benefit of using `RUBY` is that editors and github will then syntax highlight the inside of the heredoc!